### DISCUSSION TITLE
Relplot documentation doesn't mention the 'ci' parameter

### DISCUSSION THREAD

**Discussion by adgrig** [mwaskom/seaborn#3030] [Q&A] [answered]
I was just looking through https://seaborn.pydata.org/generated/seaborn.relplot.html and specifying the kind of variable to use for the 'ci' parameter of the .relplot() method seems to be missing. Will this be added in the near future to the docs/ was it removed by accident?

**Comment by mwaskom** [comment, marked as answer]
Hi, please notice how the parameter list includes `**kwargs`, which is documented as 

> kwargs: key, value pairings
>    Other keyword arguments are passed through to the underlying plotting function.

There are additional `kind`-specific parameters that are not documented in `relplot` but would be documented for either `scatterplot` or `lineplot` (`ci` appears in the docstring for the latter).

That said, please note that `ci` [has been deprecated](https://seaborn.pydata.org/whatsnew/v0.12.0.html#more-flexible-errorbars) in favor of `errorbar`.

**Reply by adgrig** [reply]
Thank you for the detailed reply!
